jscomp/bsb/bsb_ninja_file_groups.ml

```diff
@@ -126,6 +155,8 @@ let emit_module_build
       ~outputs:[output_cmi]
       ~inputs:[output_iast]
       ~rule:(if is_dev then rules.mi_dev else rules.mi)
+      ~bs_dependencies
+      ~rel_deps:(rel_bs_config_json :: relative_ns_cmi)
```

Comment:
These are the only relevant changes here. Everything else is just moving code so that it's in scope.

Reply:
When doing this at least makes it into two commits, so that it can be reviewed separatedly.